internal/server/evaluation/ofrep_bridge.go

```diff
@@ -0,0 +1,79 @@
+package evaluation
+
+import (
+	"context"
+	"errors"
+	"strconv"
+
+	"go.flipt.io/flipt/internal/server/ofrep"
+
+	rpcevaluation "go.flipt.io/flipt/rpc/flipt/evaluation"
+
+	fliptotel "go.flipt.io/flipt/internal/server/otel"
+	"go.flipt.io/flipt/internal/storage"
+	"go.flipt.io/flipt/rpc/flipt"
+	"go.opentelemetry.io/otel/trace"
+)
+
+func (s *Server) OFREPEvaluationBridge(ctx context.Context, input ofrep.EvaluationBridgeInput) (ofrep.EvaluationBridgeOutput, error) {
+	flag, err := s.store.GetFlag(ctx, storage.NewResource(input.NamespaceKey, input.FlagKey))
+	if err != nil {
+		return ofrep.EvaluationBridgeOutput{}, err
+	}
+
+	span := trace.SpanFromContext(ctx)
+	span.SetAttributes(
+		fliptotel.AttributeNamespace.String(input.NamespaceKey),
+		fliptotel.AttributeFlag.String(input.FlagKey),
+		fliptotel.AttributeProviderName,
+	)
+
+	req := &rpcevaluation.EvaluationRequest{
```

Comment:
does ofrep have the idea of an 'entityID'? if not we generate one, but just curious

Reply:
I don't think so! In this case maybe we can propose smth to the spec